### PR TITLE
[Merged by Bors] - chore: tidy various files

### PR DESCRIPTION
**Pull request by Ruben-VandeVelde**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/AlgebraicGeometry/EllipticCurve/Projective.lean [hunk 1]
```diff
@@ -286,8 +286,13 @@ lemma equiv_of_Z_eq_zero {P Q : Fin 3 → F} (hP : W.Nonsingular P) (hQ : W.Nons
     (hPz : P z = 0) (hQz : Q z = 0) : P ≈ Q := by
   rw [fin3_def P, hPz] at hP ⊢
   rw [fin3_def Q, hQz] at hQ ⊢
-  simp [nonsingular_iff, equation_iff] at hP hQ
-  simp [pow_eq_zero hP.left.symm, pow_eq_zero hQ.left.symm] at *
+  simp only [nonsingular_iff, equation_iff, Matrix.cons_val_one, Matrix.head_cons,
```
Comment (jcommelin) [inline Mathlib/AlgebraicGeometry/EllipticCurve/Projective.lean:289]: Would it make sense to preserve the original, using something like `simp says`?

File: Mathlib/AlgebraicGeometry/EllipticCurve/Jacobian.lean [hunk 2]
```diff
@@ -297,7 +297,9 @@ lemma equiv_of_Z_eq_zero {P Q : Fin 3 → F} (hP : W.Nonsingular P) (hQ : W.Nons
   have hPy : P y ≠ 0 := fun h => by simp [h] at hP; exact hPx <| pow_eq_zero hP.left.symm
   have hQy : Q y ≠ 0 := fun h => by simp [h] at hQ; exact hQx <| pow_eq_zero hQ.left.symm
   use Units.mk0 _ <| mul_ne_zero (div_ne_zero hPy hPx) (div_ne_zero hQx hQy)
-  simp [smul_fin3, mul_pow, div_pow]
+  simp only [Units.mk0_mul, smul_fin3, Units.val_mul, Units.val_mk0, mul_pow, div_pow,
```
Comment (riccardobrasca) [inline Mathlib/AlgebraicGeometry/EllipticCurve/Jacobian.lean:300]: Do we want `simp? says`?

**Review by riccardobrasca — APPROVED** [review]
Thanks!

bors d+

**Comment by Ruben-VandeVelde** [comment]
bors merge

**Pull request closed without merging**